### ISSUE TITLE
enhancement: move tiling around

### ISSUE THREAD

**Issue by Bamban** [gnunn1/tilix#1335]
Hi again,
Is it possible to imagine moving tiles around like it is done in terminator with CTRL+right click ?
When a tile is open, I found it impossible to move.

**Comment by gnunn1** [comment]
You can move terminals around by dragging the titlebar or using Alt+Left click dragging in any part of the terminal.

**Comment by Bamban** [comment]
Alt-Left click is already bound in many distributions to the action "moving the window". So in my case it moves the entire tilix window but not the tiles (and I want to keep it that way). A possible remaping ?

**Comment by gnunn1** [comment]
Ctrl is already used by the VTE to select columns, I do not have any plans to change the Alt key since it works fine in Gnome which is what I support.

**Comment by Bamban** [comment]
Let me rephrase. I don't ask you to remap (of course) but maybe giving the choice to the user. In my case, alt+right-click would be OK.

**Comment by gnunn1** [comment]
I understood, I do not have any plans to add an option for this as I'm already dealing with too many options. in tilix from a support perspective. My target desktop environment is Gnome so generally I do not go out of my way to add options for other DEs, it's just a limitation of what one person can support.

**Comment by Bubu** [comment]
Nice, after reading this I figured out that in Mate you can change the window movement key from alt to super. So working fine here now! :-)

**Comment by jpfleury** [comment]
I had this problem too. XFCE uses `Alt+Left click` to move windows. For those looking at changing it in XFCE:

1. Open the settings editor (command `xfce4-settings-editor`).
2. In the left column, choose `xfwm4`.
3. In the right pane, find the property `easy_click` and delete the value `Alt` or change it for another key.

Now `Alt+Left click` will work with Tilix.

**Comment by zeros48** [comment]
On Mint Cinnamon I could start dragging the pane by pressing `Ctrl+Alt click`, then releasing `Ctrl` allowed me to move the pane.
